setup.cfg

```diff
@@ -42,6 +42,7 @@ docs = sphinx
 select = E101,W191,W291,W293,E111,E112,E113,W292,W391
 exclude = .eggs,.git
 [tool:pytest]
+testpaths = mpmath docs
```

Comment:
What happens when this is removed?

Is it safer to allow [pytest discovery](https://docs.pytest.org/en/6.2.x/goodpractices.html#test-discovery) to find tests wherever they may be in the repo.

Reply:
> What happens when this is removed?

```
$ pip install -U -e .
...
$ pytest
...
ERROR  - _pytest.pathlib.ImportPathMismatchError: ('mpmath.tests.conftest', '/home/sk/src/mpmath/build/lib/mpmath/tests/conftest.py', PosixPath('/home/sk/src/mpmat...
...
```
This can be fixed also by `--import-mode importlib`, but not for py2.